app/src/main/java/io/appium/settings/LocationTracker.java

```diff
@@ -262,13 +269,13 @@ public synchronized Location getLocation(Context context) {
             }
             if (mLocationManager != null && mLocationProvider != null) {
                 try {
-                    mLocation = mLocationManager.getLastKnownLocation(mLocationProvider);
```

Comment:
mLocation should still be assigned and then returned

Reply:
@mykola-mokhnach done